### PR TITLE
Finish upgrading to `hyp3lib` v2

### PR DESCRIPTION
**Pull request by jtherrmann**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,13 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [7.0.0]
+
+### Changed
+- Upgraded `hyp3lib` dependency to version `2.x.x`.
+- As of [HyP3-lib v2.0.0](https://github.com/ASFHyP3/hyp3-lib/releases/tag/v2.0.0), the [Copernicus Data Space Ecosystem (CDSE)](https://dataspace.copernicus.eu/) will now be used for downloading Sentinel-1 orbit files from `ESA`.
```
Comment (jtherrmann) [inline CHANGELOG.md:13]:
```suggestion
- As of [HyP3-lib v2.0.0](https://github.com/ASFHyP3/hyp3-lib/releases/tag/v2.0.0), the [Copernicus Data Space Ecosystem (CDSE)](https://dataspace.copernicus.eu/) will now be used for downloading Sentinel-1 orbit files from ESA.
```

**Comment by forrestfwilliams** [comment]
@jtherrmann I'm not seeing any esa-password related changes to the InSAR CLI. Does that need an update as well?

File: README.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ rtc_sentinel.py S1A_IW_SLC__1SSV_20150621T120220_20150621T120232_006471_008934_7
 To run the RTC process through the HyP3 interface:
 ```
 hyp3_gamma ++process rtc --help
-hyp3_gamma ++process rtc --username ${EDL_USERNAME} --password ${EDL_PASSWORD} \
+hyp3_gamma ++process rtc --username ${EDL_USERNAME} --password ${EDL_PASSWORD} --esa-username ${ESA_USERNAME} --esa-password {$ESA_PASSWORD} \
```
Comment (jtherrmann) [inline README.md:41]:
```suggestion
hyp3_gamma ++process rtc --username ${EDL_USERNAME} --password ${EDL_PASSWORD} --esa-username ${ESA_USERNAME} --esa-password ${ESA_PASSWORD} \
```

File: README.md [hunk 3]
```diff
@@ -60,7 +60,7 @@ ifm_sentinel.py \
 To run the InSAR process through the HyP3 interface:
 ```
 hyp3_gamma ++process insar --help
-hyp3_gamma ++process insar --username ${EDL_USERNAME} --password ${EDL_PASSWORD} \
+hyp3_gamma ++process insar --username ${EDL_USERNAME} --password ${EDL_PASSWORD} --esa-username ${ESA_USERNAME} --esa-password {$ESA_PASSWORD} \
```
Comment (jtherrmann) [inline README.md:63]:
```suggestion
hyp3_gamma ++process insar --username ${EDL_USERNAME} --password ${EDL_PASSWORD} --esa-username ${ESA_USERNAME} --esa-password ${ESA_PASSWORD} \
```

File: hyp3_gamma/__main__.py [hunk 4]
```diff
@@ -58,10 +59,30 @@ def check_earthdata_credentials(username, password):
     return username, password
 
 
+def check_esa_credentials(username: Optional[str], password: Optional[str]) -> None:
```
Comment (forrestfwilliams) [inline hyp3_gamma/__main__.py:62]: My preference would be to see both this function and the `check_earthdata_credentials` in utils.py, but it's not blocking.

**Review by forrestfwilliams — APPROVED** [review]
LGTM!

**Pull request merged**